APITestingFramework/StepDefinitions/User/Delete.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Text.Json;
+using Core;
+using Features.GeneralSteps;
+using Models;
+using RestSharp;
+using TechTalk.SpecFlow;
+
+namespace Features.User.Delete
+{
+    [Binding]
+    [Scope(Feature = "User Deletion")]
+    public class DeleteStepDefinitions : CommonSteps
+    {
+        private readonly ScenarioContext _scenarioContext;
+        private readonly RestHelper client = new RestHelper("https://todo.ly/api");
```

Comment:
client just should be initialize once unless you need to point to other url

Reply:
Done.